addons/web/static/src/core/popover/popover.js

```diff
@@ -155,23 +154,31 @@ export class Popover extends Component {
         this.subPopovers = new Set();
         useSubEnv({ [POPOVER_SYMBOL]: this.subPopovers });
 
+        onMounted(() => POPOVERS.set(this.props.target, this.popoverRef.el));
+        onWillDestroy(() => {
+            POPOVERS.delete(this.props.target);
+            this.props.parentSubPopovers?.delete(this);
+        });
+
+        if (!this.props.close) {
+            return;
+        }
         if (this.props.target.isConnected) {
             useClickAway((target) => this.onClickAway(target));
 
             if (this.props.closeOnEscape) {
-                useHotkey("escape", () => this.props.close());
+                const hotkeyService = useService("hotkey");
+                const remove = hotkeyService.add("escape", this.props.close);
+                onWillDestroy(remove);
```

Comment:
I don't think escape should close it before it is open

Reply:
I am going to remove it